src/WordPress/WordPressAPI.php

```diff
@@ -69,6 +70,48 @@ public function getHostAPIKey()
     }
 
     /**
+     * We wrap the return value with an array to be consistent between
+     * other plugins.
+     * 
+     * @param null $userId
+     *
+     * @return mixed
+     */
+    public function getDomainList($userId = null)
+    {
+        $cachedDomainName = $this->dataStore->getDomainNameCache();
+        if (empty($cachedDomainName)) {
```

Comment:
Shouldn't the logic be:

1. Check cache for value.  
2. If empty set cache equal to output of `getOriginalDomain()`
3. return domain.

If for some reason the cache fails this method will return an empty array which is incorrect behavior since this method should always return a MINIMUM of one result.